edu-services/gradebook-service/api/src/java/org/sakaiproject/service/gradebook/shared/GradebookExternalAssessmentService.java

```diff
@@ -337,4 +338,14 @@ public boolean isExternalAssignmentVisible(String gradebookUid, String externalI
 	 * @return Whether the gradebook has categories enabled
 	 */
 	public boolean isCategoriesEnabled(String gradebookUid);
+
+	/**
+	 * Get the internal ID of an externally managed gradebook item by it's external ID
+	 * @param gradebookUUID the UUID of the gradebook to check
+	 * @param externalID the external ID of the gradebook item in question
+	 * @return Long value of the internal ID of the gradebook ID in question
+	 * @throws GradebookNotFoundException
+	 * @throws AssessmentNotFoundException
+	 */
+	public OptionalLong getInternalAssessmentID(String gradebookUUID, String externalID) throws GradebookNotFoundException, AssessmentNotFoundException;
```

Comment:
Assessment sounds like Samigo. Why not just call it GradebookItem? You use that terminology in the javadocs.

Reply:
In the external service, all other methods and JavaDocs refer to it as an 'assessment', as it's called the GradebookExternal*Assessment*Service.

I agree with you that I like gradebook item rather than assessment, but using that terminology in only this method is not consistent with the rest of the classes.

Reply:
@adrianfish, I've changed the JavaDocs to reference "assessment (gradebook item)". Hopefully this helps while keeping the file consistent.

Reply:
@adrianfish, can you take another look at this please?